### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by marler8997**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1 @@
+Permission is granted to use/copy/modify the contents of this repository without restriction.
```
Comment (timotheecour) [inline LICENSE]: why not just use one of the standard licenses, eg see https://github.com/dlang-community/D-Scanner/blob/master/LICENSE_1_0.txt

Comment (marler8997) [reply LICENSE]: I don't know of a standard license that is as free as what I've written here...do you?

Comment (marler8997) [reply LICENSE]: Well, there is WTFPL, but I don't want to put people off by including a "curse word" in my license.

File: archive/har.d [hunk 2]
```diff
@@ -1,3 +1,23 @@
+/**
+HAR - Human Archive Format
+
+https://github.com/marler8997/har
+
+HAR is a simple format to represent multiple files in a single block of text, i.e.
+---
```
Comment (timotheecour) [inline archive/har.d:7]: why not use ``` instead of --- for code delimiter to avoid clashing w HAR:

```
--- main.d
etc...
```

Comment (marler8997) [reply archive/har.d:7]: It doesn't clash, except visually.  Does ddoc support triple backticks?

File: archive/har.d [hunk 3]
```diff
@@ -1,3 +1,23 @@
+/**
```
Comment (timotheecour) [inline archive/har.d:1]: IMO there's almost never a case where /* should be used ; use /+  instead which allows nested comments

File: archive/har.d [hunk 4]
```diff
@@ -288,7 +307,8 @@ struct FileProperties
 
 // returns a formatter that will print the given string.  it will print
 // it surrounded with quotes if the string contains any spaces.
-@property auto formatQuotedIfSpaces(T...)(T args) if(T.length > 0)
+@property auto formatQuotedIfSpaces(T...)(T args)
```
Comment (timotheecour) [inline archive/har.d]: i agree with @wilzbach about @property being useless

File: archive/har.d [hunk 5]
```diff
@@ -297,21 +317,21 @@ struct FileProperties
         {
             import std.string : indexOf;
             bool useQuotes = false;
-            foreach(arg; args)
+            foreach (arg; args)
```
Comment (timotheecour) [inline archive/har.d:320]: always a good idea to split out in different commits formatting-only changes (eg using dfmt) vs actual code changes (unless https://github.com/dlang-community/dfmt/issues/159 is implemented)

File: archive/har.d (hunk 3 above)
Comment (marler8997) [reply archive/har.d:1]: I think I only see `/** ... */` in the compiler source...

File: dub.sdl [hunk 6]
```diff
@@ -0,0 +1,14 @@
+name "har"
+description "A human-readable archive format"
+authors "Jonathan Marler"
+copyright "Copyright © 2018, Jonathan Marler"
+license "public domain"
```
Comment (timotheecour) [inline dub.sdl]: cf https://github.com/marler8997/har/pull/5/files#r174590228

File: archive/har.d (hunk 2 above)
Comment (marler8997) [reply archive/har.d:7]: I get a ddoc error when using triple backticks

File: dub.sdl [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+targetType "executable"
+dependency "har:library" version="*"
+sourceFiles "harmain.d"
+
+subPackage {
```
Comment (timotheecour) [inline dub.sdl:10]: * how about adding subPackage for the other things (examples, test)
* using a `source` subdirectory is common practice eg:
`source/archive/har.d`

File: archive/har.d (hunk 5 above)
Comment (marler8997) [reply archive/har.d:320]: I agree in most cases, but I'm fine with it in the scenario, mostly because I'm too lazy to split the changes into different commits :)

File: dub.sdl (hunk 7 above)
Comment (marler8997) [reply dub.sdl:10]: done
